test/test_ucum.cpp

```diff
@@ -18,9 +18,9 @@ SPDX-License-Identifier: BSD-3-Clause
 TEST(UCUM, domain_exclusion)
 {
     using namespace units::domains;
-    EXPECT_EQ(ucum | astronomy, allDomains);
-    EXPECT_EQ(ucum | nuclear, allDomains);
-    EXPECT_EQ(nuclear | astronomy, allDomains);
+    // EXPECT_EQ(ucum | astronomy, allDomains);
+    // EXPECT_EQ(ucum | nuclear, allDomains);
+    // EXPECT_EQ(nuclear | astronomy, allDomains);
```

Comment:
Intentionally commented?

Reply:
I am changing the domain codes so this relationship doesn't hold at present,  there may be further modifications to restore them but not sure what the best approach is yet.